general/server/quick-connect.md

```diff
@@ -0,0 +1,16 @@
+---
+uid: server-quick-connect
+title: Quick Connect
+---
+
+# Quick Connect
+
+Starting with Jellyfin server version 10.7.0 and supported clients, you can use Quick Connect to sign in to your account without the need of a password. You need to previously be logged into a supported client, like the default Jellyfin Web Client.
+
+## Enabling Quick Connect
+
+To use Quick Connect, the Jellyfin server admin has to enable this feature in the server dashboard.
```

Comment:
```suggestion
To use Quick Connect, the Jellyfin server admin has to enable this feature in the server dashboard.

Settings > Dashboard > Quick Connect > Enable quick connect on this server (http://localhost:8096/web/index.html#!/quickConnect.html)
```

Reply:
Includes directions on how to get to the settings. URL included, but is less important IMO. Feel free to remove.